covid_moonshot/analysis/plots.py

```diff
@@ -459,6 +542,26 @@ def save_run_level_plots(
         )
         fig.suptitle(f"RUN{run}")
 
+    with save_plot(name=f"RUN{run}-bootstrapped-CLONEs"):
+
+        # Gather CLONES per GEN for run
+        clones_per_gen = min(
```

Comment:
nit: this might be a little cleaner as a single list comprehension

```python
clones_per_gen = min(
    [
        len(works)
        for phase in [solvent_phase, complex_phase]
        for gen in phase.gens
        for works in [gen.forward_works, gen.reverse_works]
    ]
)
```